quicklogic/CMakeLists.txt

```diff
@@ -1 +1,3 @@
+set(QLF_FPGA_DATABASE_DIR ${symbiflow-arch-defs_SOURCE_DIR}/third_party/qlfpga-symbiflow-plugins)
```

Comment:
It looks like this is never used. Can we remove?

Reply:
This is actually used when defining the arch and device, also to access the fasm database